### PR TITLE
Add command for automating local setup of Edly Multisite Devstack

### PR DESCRIPTION
**Pull request by mfarhan943**
**Description:**
To enhance the devstack process, we aim to automate the local setup of Edly Multisite Devstack.

**JIRA:**
[EDLY-5506](https://edlyio.atlassian.net/browse/EDLY-5506)

[EDLY-5506]: https://edlyio.atlassian.net/browse/EDLY-5506?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by taimoor-ahmed-1 — APPROVED** [review]

File: openedx/core/djangoapps/theming/management/commands/setup_edly_multisite_devstack_Juniper_locally.py [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+        Switch.objects.get_or_create(name='verify_student_disable_account_activation_requirement', active=True)
+        Switch.objects.get_or_create(name='enable_course_progress_feature', active=True)
+
+    @staticmethod
+    def update_credentials():
+        """Update credentials and redirect uris."""
+        Application.objects.filter(name='credentials-sso').update(
+            redirect_uris='http://edx.devstack.lms:18150/complete/edx-oauth2/')
+
+        CatalogIntegration.objects.get_or_create(internal_api_url='https://edx.devstack.lms:1381/api', enabled=True)
```
Comment (marslanabdulrauf) [inline openedx/core/djangoapps/theming/management/commands/setup_edly_multisite_devstack_Juniper_locally.py:216]: can you confirm the port number ? isn't it `18381` ?

Comment (mfarhan943) [reply openedx/core/djangoapps/theming/management/commands/setup_edly_multisite_devstack_Juniper_locally.py:216]: According to the Confluence document, the port is 1381.

Comment (marslanabdulrauf) [reply openedx/core/djangoapps/theming/management/commands/setup_edly_multisite_devstack_Juniper_locally.py:216]: There must be typo in confluence, please fix it there as well.

Comment (mfarhan943) [reply openedx/core/djangoapps/theming/management/commands/setup_edly_multisite_devstack_Juniper_locally.py]: Done

**Review by marslanabdulrauf — APPROVED** [review]

**Pull request merged**